.github/workflows/terratests.yml

```diff
@@ -51,11 +51,6 @@ jobs:
       
       - name: "go vet"
         run: go vet ./...
-      
-      - name: golangci-lint
```

Comment:
This fixes it


      - name: golangci-lint
        uses: golangci/golangci-lint-action@v1
        with:
          # Optional: golangci-lint command line arguments.
          args: --timeout=3m0s
          version: ${{ env.golangci_lint_version }}